03_how_ln_works.asciidoc

```diff
@@ -609,159 +621,117 @@ These fees are paid to the miner who mines that particular block.
+
+The synchronous and always-online nature of the Lightning network is probably the biggest difference in the user experience and often confounds users who are accustomed to Bitcoin.
+
+==== Satoshis vs. milli-satoshis
+
+On Bitcoin the smallest amount is a _satoshi_ which cannot be divided any further. Lightning is a bit more flexible, and Lightning nodes work with _milli-satoshis_ (thousandths of a satoshi). This allows tiny payments to be sent via Lightning. A single milli-satoshi payment can be comfortably sent across a payment channel, which based on current bitcoin value would be the equivalent of a hundredth thousandth of a US cent (or $0.00000009 USD). Less than a micropayment, this has been characterized as a _nanopayment_. Routing across multiple channels, such an amount would likely be overwhelmed by fees, but some nodes advertize zero-fee routing, so even that is currently possible.
+
+The milli-satoshi unit cannot, of course, be settled on the Bitcoin blockchain at that granularity. Upon channel closure, balances are rounded to the nearest satoshi. But over the lifetime of a channel, millions of nanopayments are possible at milli-satoshi levels. The Lightning network breaks throught the micropayment barrier.
 
 === Commonality of Bitcoin and Lightning
 
 ==== Monetary Unit
 
-Both the Bitcoin network and the Lightning network use the same monetary units: bitcoins. 
-Lightning payments use the very same bitcoins as Bitcoin transactions.
-As an implication, because the monetary unit is the same, also the economic monetary limit is the same: 21 million bitcoins.
-From these 21 million bitcoins, some are locked in 2-out-of-2 multisig addresses so that they can enable a payment channel on the Lightning Network.
+Both the Bitcoin network and the Lightning network use the same monetary units: bitcoin. Lightning payments use the very same bitcoin as Bitcoin transactions. As an implication, because the monetary unit is the same, the monetary limit is the same: less than 21 million bitcoin. Of Bitcoin's 21 million total bitcoin, some are already allocated to 2-of-2 multi-signature addresses as part of payment channel on the Lightning Network.
+
+==== Irreversibility and finality of payments
+
+Both Bitcoin transactions and Lightning payments are irreversible and immutable. There is no "undo" operation or "chargeback" for either system. As a sender of either one you have to act responsibly, but also, yas a recipient you are guaranteed finality of your transactions.
+
+==== Trust and counterparty risk
 
-==== Irrevocability 
+Just as Bitcoin, Lightning requires the user only to trust mathematics, encryption, and that the software does not have any critical bugs. Neither Bitcoin nor Lightning require the user to trust a person, a company, an institution, or a government.
+Since Lightning sits on top of Bitcoin and relies on Bitcoin as its underlying base layer, it is clear that the security model of Lightning reduces to the security of Bitcoin.
```

Comment:
maybe: The security of Lightning can never be better than the one of Bitcoin but only worse (as it is en extension of bitcoin)